src/platform/src/test/java/org/geoserver/platform/resource/FileWrapperResourceTheoryTest.java

```diff
@@ -14,6 +14,15 @@
 import org.junit.experimental.theories.DataPoints;
 import org.junit.rules.TemporaryFolder;
 
+/**
+ * Test handling of file wrapper (Files.ResourceAdaptor) which works with absolute paths internally.
+ *
+ * <p>This ResourceAdaptor is intended to allow GeoServer to work with absolute paths outside of the
+ * data direct. As a result stores an absolute file internally.
+ *
+ * <p>As ResourceTheoryTest covers a lot of relative paths {@link #getResource()} has been adjusted
+ * to be forgiving when absolute paths are used.
```

Comment:
```suggestion
```

I adjusted the ResourceTheoryTest so broken absolute path would not be generated, as a result this comment no longer applies.